### ISSUE TITLE
Chart info (e.g. # crossovers) inaccurate in pump mode

### ISSUE THREAD

**Issue by aeubanks** [dguzek/Simply-Love-SM5#411]
I have a bunch of autogen pump charts that don't have any crossovers, yet on the song select screen it says there are a bunch of crossovers/sideswitches/footswitches/etc.

**Comment by teejusb** [comment]
The pattern analyzer I don't think works for anything but 4 panel in general

**Comment by aeubanks** [comment]
So those stats shouldn't be shown (or be "?") in non-4 panel modes?

**Comment by teejusb** [comment]
Yeah I think that would make sense. Ideally the algo would work regardless of #panels but it's not generic enough I think.
